src/transformers/models/speech_encoder_decoder/modeling_speech_encoder_decoder.py

```diff
@@ -176,7 +181,7 @@ class SpeechEncoderDecoderModel(PreTrainedModel):
     """
     config_class = SpeechEncoderDecoderConfig
     base_model_prefix = "speech_encoder_decoder"
-    main_input_name = "input_values"
+    main_input_name = "inputs"
```

Comment:
SpeechEncoderDecoder is can have multiple different input formats -> change main input name to `inputs`. This has no breaking changes

Reply:
Just like `perceiver`